ci/lib.py

```diff
@@ -166,7 +166,8 @@ def generate_ansible_inventory(jenkins_master_host, jenkins_slave_host,
 log_level=DEBUG
 openshift_server_ip={openshift_host}
 deployment=ci
-cccp_index_repo=https://github.com/rtnpro/container-index.git
+cccp_index_repo=https://github.com/navidshaikh/cccp-index.git
```

Comment:
I would say lets make it `https://github.com/centos/container-index.git` with CI branch

Reply:
that was purposefully added to have the PR tested in CI if it works with a git repo and a branch, will replace the repo URL in another commit.